cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -116,9 +46,37 @@
     .o-expandable_header-right {
         width: 27%;
     }
+
+    .respond-to-max( @bp-xs-max, {
+        overflow: hidden;
+    } );
 }
 
 .org-chart_branch {
+
+    &_name {
+        position: relative;
+
+        .respond-to-min( @bp-sm-min, {
+            margin: @margin / 2 0;
```

Comment:
Minor, I'd wrap `@margin / 2` in parens just so it's clear that's the top/bottom and the 0 is left right. It should also probably be converted to `em`s